test/hotspot/jtreg/applications/jcstress/JcstressRunner.java

```diff
@@ -104,11 +108,26 @@ private static String[] getCmd(String[] args) {
 
         extraFlags.add("--jvmArgs");
         extraFlags.add("-Djava.io.tmpdir=" + System.getProperty("user.dir"));
+
+        // The "default" preset might take days for some tests
+        // so use sanity testing by default.
+        String mode = "sanity";
```

Comment:
`sanity` mode is incorrect for actual testing runs. Should be at least `quick`.

Reply:
fixed